### ISSUE TITLE
[archive] opensource: CV open-source radar 2026-09-18

### ISSUE THREAD

**Issue by Jung-woojin** [Jung-woojin/cv-research-radar#2]
# CV 오픈소스 레이더 — 2026-09-18

관측 범위는 2026-09-11~2026-09-18. 단순 검색 결과의 상대 시각이 아니라 Git commit, PyPI release, Hugging Face update, 실제 code release 시각을 중심으로 확인했다. 논문은 이번 주에 나왔지만 코드가 이전부터 공개돼 있던 경우는 신규 오픈소스로 세지 않았다.

## 이번 주 바로 clone할 후보

| 우선 | 프로젝트 | 이번 주 변화 | 실험 가치 | 재현성 | 신규성 | 실용성 |
|---|---|---|---:|---:|---:|---:|
| 1 | D-FINE-seg v0.4.2 | 9/16 C++ TensorRT + GPU video pipeline + PyPI | 5.0 | 4.8 | 4.1 | 5.0 |
| 2 | RelateAnything | 9/11 논문, 9/14 release 정리·모델/문서 대규모 업데이트 | 4.9 | 4.8 | 5.0 | 4.8 |
| 3 | VPRef | 9/15 논문, 9/16 코드 공개, 9/18까지 업데이트 | 4.6 | 3.0 | 4.6 | 3.6 |
| 4 | OPD-Aha 4B/9B | 9/15 논문 + 공식 HF weights 공개/업데이트 | 4.7 | 4.4 | 4.7 | 3.2 |
| 5 | PANORAMA + PanoCaps | 9/16 논문, 9/17 코드 + 3개 dataset 공개 | 4.8 | 3.6 | 4.9 | 3.0 |

네 장비에서 오늘 바로 실험한다는 조건까지 넣으면 D-FINE-seg와 RelateAnything이 가장 현실적이다. VPRef와 PANORAMA는 full reproduction보다 benchmark/data/evaluator 활용 가치가 더 높고, OPD-Aha는 BF16 공식 모델을 8GB급 GPU에 그대로 올리기엔 무겁다.

---

## 1. D-FINE-seg v0.4.2

9월 16일 v0.4.2가 실제 release됐다. 핵심은 새로운 detection block보다 **배포 stack이 크게 완성됐다는 점**이다.

- 독립형 C++ TensorRT inference library
- detection / instance segmentation / semantic segmentation 지원
- NVDEC → inference → GPU overlay → NVENC end-to-end video pipeline
- SAM3 box-only detection path
- PyPI `dfine-seg 0.4.2`
- DDP / EMA / AMP / OneCycle / WandB / ByteTrack / SAM3 auto-labeling
- ONNX / TensorRT / OpenVINO / CoreML / LiteRT export

N/S/M/L/X 크기의 COCO-pretrained weights를 제공한다. 저자 benchmark 기준 Cityscapes S detection 모델은 10.3M params, F1 0.703, RTX 5070 Ti TensorRT FP16 end-to-end 약 2.0ms다. 이 수치는 독립 검증 SOTA라기보다 제작자 benchmark로 해석해야 하지만, aerial detector의 deployment baseline으로는 상당히 매력적이다.

**라이선스:** Apache-2.0  
**호환성:** MMDetection plugin이나 Ultralytics API 직접 호환은 아니지만 YOLO-format annotation, COCO JSON, ONNX/TensorRT export 때문에 기존 pipeline과 연결하기 쉽다.  
**GPU 현실성:** 공식 최소 VRAM 표는 없지만 N/S 모델, AMP, 작은 batch, gradient accumulation을 이용하면 8GB급 GPU에서 가장 현실적으로 training까지 시도할 후보.

바로 해볼 실험은 **VisDrone 또는 자체 aerial dataset에서 S model fine-tune → Ultralytics YOLO baseline과 accuracy/latency 동일 조건 비교**다.

- GitHub: https://github.com/ArgoHA/D-FINE-seg
- Hugging Face: https://huggingface.co/ArgoSA/D-FINE-seg

---

## 2. RelateAnything

정확한 제목은 **“RelateAnything: Real-Time Open-Vocabulary Relation Prediction From Any Inputs”**. 9월 11일 공개.

기존 OVD가

```
(image, class vocabulary) -> boxes
```

라면 RelateAnything은

```
(image, regions, relation vocabulary) -> relations
```

으로 문제를 바꾼다.

object class label을 relation model에 넣지 않고, image와 region만 받아 inference-time arbitrary predicate vocabulary를 사용한다. 예를 들어 maritime 영상에서 detector가 ship A/B의 세부 class를 몰라도:

- approaching
- overtaking
- tethered to
- about to collide with
- behind

같은 predicate를 추론할 수 있다.

공식 API에서도 `model.set_vocabulary([...])`로 바로 바꿀 수 있다.

### 코드 품질

이번 주 후보 중 최고 수준에 가깝다.

- training recipe
- RA-4M generation
- OV-SGG-Bench
- release tooling
- CPU tests
- deployment scripts
- ONNX
- browser WebGPU/WASM demo
- pitfalls 문서

까지 갖춰져 있다.

추천 모델은 `relsgg-vits16plus`, 약 53.2M params. 46.1M / 53.2M / 113.8M 모델이 공개됐다. 저자 A40 batch-1 측정은 약 19–20ms.

**호환성:** image + boxes/masks 형태라 Ultralytics YOLO, MMDetection, Grounding DINO, SAM, 자체 detector 출력에 adapter 하나만 붙이면 된다.

**라이선스 주의:** 코드 자체는 Apache-2.0이지만 weights는 Meta DINOv3 license, RA-4M annotation은 Gemma Terms, demo 일부 detector는 Ultralytics AGPL-3.0 영향을 받는다. repo가 Apache라고 전체 artifact가 Apache인 것은 아니다.

연구 아이디어 관점에서는 D-FINE보다 오히려 이 repo가 더 재미있다. detector를 다시 학습하지 않고 **open-vocabulary relation reasoning layer**를 올릴 수 있기 때문이다.

- GitHub: https://github.com/SHTUPLUS/RelateAnything

---

## 3. VPRef

논문: **“VPRef: A Cross-Domain Benchmark for Referring Remote Sensing Image Segmentation”**, 9월 15일 공개. 코드 공개는 9월 16일.

Vaihingen-Potsdam 기반 **46,972 image-language-mask triplet**과 3단계 linguistic hierarchy를 만들고, visual domain drift와 textual logic drift를 동시에 다룬다. SAM3+LoRA baseline은 foundation model 전체의 1.08%만 변경한다.

이번 주 공개물 중 **aerial/UAV + multimodal grounding**과 가장 직접적으로 맞는다.

```
satellite image + free-form referring expression -> target mask
```

이라는 문제라 aerial detection에서 VLM/grounding으로 확장할 때 benchmark 구조를 참고하기 좋다.

### 단점

repo root에는 실제 코드가 올라왔지만 packaging이 D-FINE/RelateAnything 수준은 아니다.

- requirements/environment 문서가 부족
- one-command reproduction 부족
- 많은 commit이 파일 업로드 위주
- 논문 fine-tuned checkpoint가 명확히 surface되지 않음
- dataset은 별도 다운로드

또 README badge는 Apache 2.0인데 실제 LICENSE 파일은 MIT로 보여 license metadata가 일치하지 않는 문제가 있었다. 실제 사용 전 재확인이 필요하다.

8GB 환경이라면 **dataset/protocol을 먼저 받아 기존 Grounding/SAM baseline과 split을 재현**하는 것이 낫다.

- GitHub: https://github.com/quanweiliu/VPRef

---

## 4. OPD-Aha

논문: **“OPD-Aha: From Linguistic Momentum to Visual Reflection in Multimodal On-Policy Distillation”**, 9월 15일.

핵심은 잘못된 visual reasoning prefix가 누적될 때 teacher조차 linguistic momentum에 끌리는 문제를, **같은 teacher의 real-image vs visual-null distribution 차이**로 분리해 visual corrective signal을 만드는 것이다.

### 공개 상태

- 4B / 9B 공식 checkpoint
- verl 기반 multi-node training
- Ray
- FSDP checkpoint merge
- vLLM serving
- fine-grained perception evaluation
- MathVista/MathVerse/WeMath/MathVision/DynaMath evaluation

Inference는 Hugging Face image-text-to-text pipeline과 호환된다.

하지만 training reproduction은 무겁다. 공식 예제 자체가 multi-node/multi-GPU를 전제로 한다. 4B라고 이름 붙어도 실제 BF16 weight footprint는 8GB급 GPU에서 그대로 쓰기 어렵다.

따라서 로컬에서는 **checkpoint inference 또는 quantized community build로 mechanism 확인**, full training reproduction은 별도 GPU 자원이 있을 때가 맞다.

- GitHub: https://github.com/Echochef/OPD-Aha
- HF 4B: https://huggingface.co/CewEhao/OPD-Aha-4B
- HF 9B: https://huggingface.co/CewEhao/OPD-Aha-9B

---

## 5. PANORAMA + PanoCaps

논문: **“PANORAMA: Panoptic Grounded Captioning via Mask Proposal Selection”**, 9월 16일. 실제 code commit은 9월 17일.

VLM caption과 pretrained segmenter의 mask proposal pool을 연결해 **full-scene caption의 거의 모든 noun phrase를 pixel mask로 grounding**한다.

PanoCaps:
- 3,470 images
- 34K panoptic regions
- 약 17.9K unique phrases
- 약 99% pixel coverage

foreground thing뿐 아니라 background stuff까지 natural-language phrase와 연결한다.

### 재현성

training/fine-tuning/evaluation 코드는 공개됐지만 full training은 무겁다.

- Python 3.11
- torch 2.6.0 + CUDA 12.4
- transformers 4.57.1
- xTuner
- MMEngine
- DeepSpeed
- optional FlashAttention
- Java/SPICE
- Qwen3-VL-4B
- SAM3

논문 training은 16 GPU, PanoCaps evaluation script 기본값은 4 GPU.

당시 공식 HF organization에는 dataset은 공개됐지만 pretrained PANORAMA model이 명확히 노출되지 않았다. 따라서 현재 가장 현실적인 사용 방식은 **PanoCaps evaluator와 dataset을 가져와 자체 grounding/VLM을 평가**하는 것.

**라이선스:** CC BY-NC 4.0 계열이라 연구/비상업용 중심으로 봐야 한다.

- GitHub: https://github.com/sarapieri/panorama_grounding
- Dataset: https://huggingface.co/datasets/Panorama-grounding/PanoCaps

---

## 시스템 쪽: Low Precision FlashAttention-4

PyTorch가 9월 16일 **“Low Precision Flash Attention 4: End-to-End Block-Scaled Attention for Blackwell”**을 공개.

MXFP8 forward/backward와 producer-side quantization fusion, jagged tensor dataflow까지 묶어 설계했다. 주력 path는 Blackwell SM100/CUDA/CUTLASS 계열을 전제로 하므로 일반 CV 연구자가 바로 pip 설치해서 쓰는 update는 아니다.

VLM pretraining/system을 직접 할 때 참고할 항목이지 로컬 CV 연구 clone 우선순위에는 넣지 않았다.

- Blog: https://pytorch.org/blog/low-precision-flash-attention-4-end-to-end-block-scaled-attention-for-blackwell/
- Code: https://github.com/facebookresearch/ads_model_kernel_library

---

### Hyper-RED: Scalable Event Pre-training via Semantic Hypergraph Distillation

논문은 9월 15일 공개됐고 RGB→event representation transfer를 token-wise feature matching 대신 semantic hypergraph relation distillation으로 바꾸는 아이디어는 좋다.

하지만 당시 공식 GitHub는 사실상 **“The Code is coming.”** 상태였다.

즉 이건 **읽을 논문이지 clone할 repo는 아님**.

---

### LAF-YOLOv10
논문은 이번 주지만 연결된 implementation의 핵심 commit은 훨씬 이전이라 이번 주 신규 repo로 세지 않았다. 다만 여러 small-object module을 조합했더니 baseline보다 크게 떨어지는 negative result는 읽을 가치가 높다.

### SparseQID
논문은 이번 주지만 code repo는 이전부터 공개돼 있었다. QUICKSTART, test, checkpoints까지 갖춰져 multi-camera 3D tracking reference implementation으로는 좋다.

### VideoMM
paper는 이번 주지만 GitHub update가 엄격한 7일 cutoff보다 조금 앞서 신규 repo 목록에서는 제외했다.

---

## 이번 주 실제 행동 순서

1. **D-FINE-seg** — UAV/small-object/edge baseline + TensorRT deployment
2. **RelateAnything** — detector output 위 open-vocabulary relation reasoning
3. **VPRef** — aerial multimodal grounding benchmark/protocol
4. **OPD-Aha** — official checkpoint inference
5. **PanoCaps** — evaluator/dataset 활용

이번 주 흐름에서 가장 눈에 띈 건 새 detector architecture 자체보다 **기존 perception output을 open-vocabulary relation·grounding·reasoning으로 확장하는 흐름**과 **research code를 실제 deployable artifact로 완성하는 흐름**이었다.
